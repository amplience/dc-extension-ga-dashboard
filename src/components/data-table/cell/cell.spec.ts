import { render } from '@testing-library/svelte';
import Cell from './cell.svelte';

describe('Cell', () => {
  it('should render the DataTable Cell component', () => {
    const { container } = render(Cell, {});

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render the DataTable Cell component with a set width', () => {
    const { container } = render(Cell, { width: '40%' });

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render the DataTable Cell component with a set alignment', () => {
    const { container } = render(Cell, { align: 'flex-end' });

    expect(container.firstChild).toMatchSnapshot();
  });
});
