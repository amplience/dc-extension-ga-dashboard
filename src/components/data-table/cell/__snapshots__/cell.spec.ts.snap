// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell should render the DataTable Cell component 1`] = `
<div>
  <div
    class="data-table-cell svelte-1x8v3rk"
    style="
  max-width:100%;
  justify-content:flex-start;
  color:#333;"
  />
</div>
`;

exports[`Cell should render the DataTable Cell component with a set alignment 1`] = `
<div>
  <div
    class="data-table-cell svelte-1x8v3rk"
    style="
  max-width:100%;
  justify-content:flex-end;
  color:#333;"
  />
</div>
`;

exports[`Cell should render the DataTable Cell component with a set width 1`] = `
<div>
  <div
    class="data-table-cell svelte-1x8v3rk"
    style="
  max-width:40%;
  justify-content:flex-start;
  color:#333;"
  />
</div>
`;
